packages/angular/tsconfig.cjs.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./tsconfig.json",
+
+  "compilerOptions": {
+    "module": "commonjs",
```

Comment:
We should lint against this structure somehow, but we can open another ticket to track that.

Reply:
👍🏻